### PR TITLE
Setup project with lint and gitignore

### PR DESCRIPTION
**Pull request by koosoku**

### CODE REVIEW

File: scripts/config.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    apiKey: '84XAr4Z2Z57+ieH6qwuIrc6zVxnlnEm/e3BnwOW72kvwebARGqXFvKE9sa/r/cJEFRHSqXCHZ9J5gkCXCQJzDNAHEsXevE/wPzgECF1YkAUmh9hoCxmqqQGlb9aF62mhEeZHHUEaqzAqux6TSuYntWu2iQC7KF4ml1G/7UagleZ0eCKoGZmTHw=='
-}
\ No newline at end of file
+    apiKey: "84XAr4Z2Z57+ieH6qwuIrc6zVxnlnEm/e3BnwOW72kvwebARGqXFvKE9sa/r/cJEFRHSqXCHZ9J5gkCXCQJzDNAHEsXevE/wPzgECF1YkAUmh9hoCxmqqQGlb9aF62mhEeZHHUEaqzAqux6TSuYntWu2iQC7KF4ml1G/7UagleZ0eCKoGZmTHw=="
```
Comment (BrandonDanis) [inline scripts/config.js]: 🤔

Comment (koosoku) [reply scripts/config.js]: @silverAndroid

File: scripts/reviews_scraper.js [hunk 2]
```diff
@@ -1,45 +1,50 @@
...
     }
 
     do {
         try {
             const fileName = getPath(recipeID, pageNumber);
+
+            // eslint-disable-next-line prefer-destructuring
             reviews = (await getReviews(recipeID, pageNumber, pageSize)).data.reviews;
-            for (let i = 0; i < reviews.length; i++) {
+            for (let i = 0; i < reviews.length; i += 1) {
```
Comment (BrandonDanis) [inline scripts/reviews_scraper.js]: why no ++?

Comment (BrandonDanis) [reply scripts/reviews_scraper.js]: Does the linting demand this?

Comment (koosoku) [reply scripts/reviews_scraper.js]: yee

Comment (silverAndroid) [reply scripts/reviews_scraper.js]: although we could turn it off so it doesn't require it

File: scripts/config.js (hunk 1 above)
Comment (silverAndroid) [reply scripts/config.js]: This API key isn't given from them to us so it doesn't matter and doesn't count towards some kind of quota against us. It's something we would have to change each time anyways.

**Pull request merged**